Core/metadata.h

```diff
@@ -202,6 +202,8 @@ enum EMdInputOutput {
     EMioPipeIn,             // normal user-streamed input data: attributes, varyings, etc.
     EMioVertexId,
     EMioInstanceId,
+    EMioVertexIndex,
```

Comment:
I don't think this should be added here.

Can't this be entirely handled by adding the built-ins?  Note that list is much larger than this list, and generally, EMioPipeIn and EMioPipeOut are metacategories of a large number of built-ins.  This is the new trend, that the built-ins were added as subcategories of things in this list, and the only reason something might be in both lists is totally for backward compatibility reasons, but that wouldn't apply to something new.

(That is, it certainly doesn't make sense to have them in both lists.)

See comments in glslang's regarding TStorageQualifier and TBuiltInVariable for more context.

If correct, we should document EMioPipeIn/out as super categories, as in glslang.